feat(frontend): codegen ts-client + vue-client (#2142) (#2143)
ignite/cli: docs/cli/index.md

* [ignite generate](#ignite-generate)	 - Generate clients, API docs from source code

## ignite generate typescript

> Generate Typescript client for you chain's frontend from your config.yml

Reviewer: ```suggestion
Generate Typescript client for your chain's frontend from the `config.yml` file
```
